### PR TITLE
Feat/actions history

### PR DESCRIPTION
**Pull request by tobi-bams**
This PR contains the `actions_history` model and `save_history` endpoint

### CODE REVIEW

**Comment by kevkevinpal** [comment]
looking good, might want to remove the bits of code that change the paths to whats on your local machine.

We can keep it on the branch for now till we test to see if its working with the whole flow

**Comment by tobi-bams** [comment]
> looking good, might want to remove the bits of code that change the paths to whats on your local machine.
> 
> We can keep it on the branch for now till we test to see if its working with the whole flow

So if I am getting you correctly, you want to change stuffs like, the path to the LND macaroon, tls cert, and DB right??

File: config/app.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "development": {
     "senza_url": "http://localhost:3000/api/v2",
-    "macaroon_location": "/Users/evanfeenstra/code/lnd-dev/alice/data/chain/bitcoin/simnet/admin.macaroon",
-    "tls_location": "/Users/evanfeenstra/Library/Application Support/Lnd/tls.cert",
+    "macaroon_location": "/Users/mac/Desktop/work/sphinx/sphinx-stack/lnd/alice/.lnd/data/chain/bitcoin/regtest/admin.macaroon",
+    "tls_location": "/Users/mac/Desktop/work/sphinx/sphinx-stack/lnd/alice/.lnd/tls.cert",
```
Comment (kevkevinpal) [inline config/app.json:5]: these changes don't need to be inlcuded

File: testing/stack/alice-db.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "development": {
     "dialect": "sqlite",
-    "storage": "/Users/evanfeenstra/code/sphinx/sphinx-stack/relay/db/alice.db"
+    "storage": "/Users/mac/Desktop/work/sphinx/sphinx-stack/relay/db/alice.db"
```
Comment (kevkevinpal) [inline testing/stack/alice-db.json:4]: this doesnt need to be included

File: testing/stack/alice.json [hunk 3]
```diff
@@ -3,8 +3,8 @@
     "node_ip": "127.0.0.1",
     "lnd_ip": "127.0.0.1",
     "lnd_port": "10009",
-    "tls_location": "/Users/evanfeenstra/code/sphinx/sphinx-stack/lnd/alice/.lnd/tls.cert",
-    "macaroon_location": "/Users/evanfeenstra/code/sphinx/sphinx-stack/lnd/alice/.lnd/data/chain/bitcoin/regtest/admin.macaroon",
+    "tls_location": "/Users/mac/Desktop/work/sphinx/sphinx-stack/lnd/alice/.lnd/tls.cert",
+    "macaroon_location": "/Users/mac/Desktop/work/sphinx/sphinx-stack/lnd/alice/.lnd/data/chain/bitcoin/regtest/admin.macaroon",
```
Comment (kevkevinpal) [inline testing/stack/alice.json:7]: this can also be reverted

**Review by kevkevinpal — COMMENTED** [review]
Added comments on what we can change back

**Comment by kevkevinpal** [comment]
> > looking good, might want to remove the bits of code that change the paths to whats on your local machine.
> > We can keep it on the branch for now till we test to see if its working with the whole flow
> 
> So if I am getting you correctly, you want to change stuffs like, the path to the LND macaroon, tls cert, and DB right??

Commented above what we can change back

File: config/app.json (hunk 1 above)
Comment (tobi-bams) [reply config/app.json:5]: Alright, I would do that now..

**Comment by Evanfeenstra** [comment]
You may want to add the table in `utils/migrate.ts` also

**Comment by tobi-bams** [comment]
@Evanfeenstra I added the table in `utils/migrate.ts`

**Comment by Evanfeenstra** [comment]
Great job making sure its strongly typed and multitenant-friendly @tobi-bams

**Pull request merged**